newIDE/app/src/EventsBasedBehaviorEditor/EventsBasedBehaviorPropertiesEditor.js

```diff
@@ -838,7 +838,7 @@ export default function EventsBasedBehaviorPropertiesEditor(props: Props) {
                     <FlatButton
```

Comment:
And I wonder if this should even be a whole component `ResponsiveFlatButton`? It would take a mandatory icon and hide the label on small or smaller devices

Reply:
Yep! We already have a `ResponsiveRaisedButton` I'll create its sister!